File: 프로그래머스/unrated/181942. 문자열 섞기/문자열 섞기.cs
using System;
using System.Text;

public class Solution {
    public string solution(string str1, string str2) 
    {
        var sb = new StringBuilder();
        for(int i = 0; i < str1.Length; ++i)
        {
            sb.Append(str1[i]);
            sb.Append(str2[i]);
        }
        return sb.ToString();
    }
}

File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.cpp
#include <string>
#include <vector>
#include <sstream>

using namespace std;

string solution(string str1, string str2) {
    
    std::ostringstream oss;
    
    for (int i = 0; i < str1.length(); ++i)
        oss << str1[i] << str2[i];
    
    return oss.str();
}